runtime/src/block_cost_limits.rs

```diff
@@ -42,6 +42,11 @@ lazy_static! {
         (secp256k1_program::id(), COMPUTE_UNIT_TO_US_RATIO * 24),
         (ed25519_program::id(), COMPUTE_UNIT_TO_US_RATIO * 24),
         (system_program::id(), COMPUTE_UNIT_TO_US_RATIO * 5),
+        (compute_budget::id(), COMPUTE_UNIT_TO_US_RATIO * 5),
+        (solana_address_lookup_table_program::id(), COMPUTE_UNIT_TO_US_RATIO * 25),
```

Comment:
I don't see the lookup table in the numbers you provided. And the compute budget cost doesn't seem to match the calculation used for the bpf loading costs.

It looks like you compared time relative to system program, then multiplied by system program cost: `COMPUTE_UNIT_TO_US_RATIO * 5`.

Could you explain here how you got these numbers? (no need for code comment, just a response in PR)

Reply:
yea sure. I took following steps: